extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.bindings.js

```diff
@@ -0,0 +1,87 @@
+angular.module('PaperUI.controllers.configuration.bindings', [ 'ngRoute' ]).config([ '$routeProvider', function($routeProvider) {
+    // Configure the routes for this controller
+    $routeProvider.when('/configuration/bindings', {
```

Comment:
I would make the argument the other way round: Defining the routes with the controller code binds the controller to the specific URL. You can not use the same controller (code) in a different situation behind another route. Imho the routes should be defined by the application which ties all the controllers together.